.github/workflows/ci.yml

```
@@ -40,9 +40,9 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --target=${{ matrix.TARGET }}
+          args: --target=${{ matrix.TARGET }} --all-features
```

```suggestion
          args: --target=${{ matrix.TARGET }}
```
The `std` feature cannot be compiled on thumb targets so this cannot be added here.